<!--
To change the arguments when running through command line type:
      roslaunch dbot_ros pole_tracker.launch tracker_type:=particle pole_angle_topic:=pole_angle
-->

<launch>

   <!-- for particle filter tracking: particle
        for gaussian filter tracking: gaussian -->
   <arg name="tracker_type" default="particle"/>
   <arg name="pole_angle_topic" default="pole_angle"/>
   <arg name="pole_angle_offset" default="0.0"/>

   <node name="pole_tracker" pkg="dbot_ros" type="pole_tracker" output="screen"
         args="$(arg tracker_type) $(arg pole_angle_topic) $(arg pole_angle_offset)"/>
</launch>
